1-org/envs/shared/variables.tf

```diff
@@ -193,3 +193,9 @@ variable "tfc_org_name" {
   type        = string
   default     = ""
 }
+
+variable "auto_create_network" {
+  description = "Control whether the 'default' network will be created in the project."
+  type        = bool
+  default     = true
+}
```

Comment:
this should not be a variable, it is a hard coded value when using the project factory method